src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/xunitrunner-perf.py

```diff
@@ -144,12 +179,20 @@ def _copy_package_files(assembly_list, build_drop, test_location, coreroot_locat
         raise
 
 # does perf-specific tasks; converts results xml to csv and then to json, populates machine information and uploads json
-def post_process_perf_results(settings, results_location, workitem_dir):
+def post_process_perf_results(settings, results_location, workitem_dir, xunit_test_type):
     # Use the xunit perf analysis exe from nuget package here
     log.info('Converting xml to csv')
     payload_dir = fix_path(os.getenv('HELIX_CORRELATION_PAYLOAD'))
-    perf_analysis_version = (next(os.walk(os.path.join(payload_dir, 'Microsoft.DotNet.xunit.performance.analysis')))[1])[0]
-    xmlconvertorpath = os.path.join(*[payload_dir, 'Microsoft.DotNet.xunit.performance.analysis', perf_analysis_version, 'tools', 'xunit.performance.analysis.exe'])
+    xmlconvertorpath = ''
+
+    if xunit_test_type == xunit.XUNIT_CONFIG_PERF_WINDOWS:
+        perf_analysis_version = (next(os.walk(os.path.join(payload_dir, 'Microsoft.DotNet.xunit.performance.analysis')))[1])[0]
+        xmlconvertorpath = os.path.join(*[payload_dir, 'Microsoft.DotNet.xunit.performance.analysis', perf_analysis_version, 'tools', 'xunit.performance.analysis.exe'])
+    elif xunit_test_type == xunit.XUNIT_CONFIG_PERF_LINUX:
+        perf_analysis_version = (next(os.walk(os.path.join(payload_dir, 'Microsoft.DotNet.xunit.performance.analysis.cli')))[1])[0]
```

Comment:
This line is quite hard to follow in my head... What is it actually returning?  is there only a single file/forlder there (since otherwise, walk is going to return the files in whatever order it wants).  What are the [1] and [0] returning (it's hard to tell with the 4 parenthesis all squished together at the end there.  This should probably be broken up into 3-4 lines with good variable names (or put into a function) to make it clear what's actually going on here.